app/src/main/res/layout/activity_movie_detail.xml

```diff
@@ -16,252 +17,46 @@
-
-                    <TextView
-                            style="@style/MovieDetailInfoError"
-                            android:layout_width="wrap_content"
-                            android:layout_height="wrap_content"
-                            android:text="@string/error_not_available"
-                            app:visible="@{viewmodel.hasNotTrailers}"
-                            tools:visibility="visible"/>
-
-                </LinearLayout>
-
-                <LinearLayout
-                        android:id="@+id/reviewSectionContainer"
-                        android:layout_width="match_parent"
-                        android:layout_height="wrap_content"
-                        android:orientation="vertical">
-
-                    <TextView
-                            style="@style/MovieDetailTitleStyle"
-                            android:layout_width="match_parent"
-                            android:layout_height="wrap_content"
-                            android:layout_marginBottom="8dp"
-                            android:text="@string/section_reviews" />
-
-                    <ProgressBar
-                            android:layout_width="@dimen/progress_bar_detail_width"
-                            android:layout_height="@dimen/progress_bar_detail_height"
-                            android:layout_gravity="center"
-                            app:visible="@{viewmodel.loadingReviews}"/>
-
-                    <androidx.recyclerview.widget.RecyclerView
-                            android:id="@+id/reviewListView"
-                            android:layout_width="match_parent"
-                            android:layout_height="wrap_content"
-                            app:items="@{viewmodel.reviews}"
-                            app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
-                            tools:listitem="@layout/item_review"
-                            tools:itemCount="1"/>
-
-                    <TextView
-                            style="@style/MovieDetailInfoError"
-                            android:layout_width="wrap_content"
-                            android:layout_height="wrap_content"
-                            android:text="@string/error_not_available"
-                            app:visible="@{viewmodel.hasNotReviews}"
-                            tools:visibility="visible"/>
-
-                </LinearLayout>
+                    android:paddingBottom="4dp"
+                    android:paddingEnd="8dp"
+                    android:paddingStart="8dp"
+                    android:paddingTop="4dp">
+
+                <include
+                        layout="@layout/section_movie_info"
+                        bind:movie="@{viewmodel.infoMovie}"
+                        bind:movieDetail="@{viewmodel.movieDetail}"/>
+
+                <include
+                        layout="@layout/section_keyword_list"
```

Comment:
Are you familiar with custom views?

For theses views that you will implement multiple times, you could use it.

Maybe you can create a class that extends from CardView, or layout, and receive in the constructor the values you need to  create your section:
* title seccion
* the list of elements
* error keyword

<androidx.cardview.widget.CardView
            ...>
        <LinearLayout
                ...>
            <TextView
                    ...
                    android:text="[title seccion]" />
            <ProgressBar
                    ...
                    app:visible="@{loading}" />
            <androidx.recyclerview.widget.RecyclerView
                    ... />
            <TextView
                    ...
                    app:visible="[error keyword]" />
        </LinearLayout>
    </androidx.cardview.widget.CardView>

Reply:
I'm still thinking how movie detail's view is going to look. For now, I prefer to handle the views in that way before experimenting with custom views.